src/utils.mjs

```diff
@@ -313,7 +313,7 @@ export function extractAdobeRoutingInfo(value) {
       return acc;
     }, {});
   if (routingInfo.ams) {
-    return `${routingInfo.ams}.adobecqms.net`;
+    return `${routingInfo.tier}.${routingInfo.env}.${routingInfo.ams}.adobecqms.net`;
```

Comment:
```suggestion
    return `${routingInfo.tier}-${routingInfo.env}-${routingInfo.ams}.adobecqms.net`;
```

I know, it's a bit unreasonable, as these are not real hostnames, and we don't have real traffic going there, but multi-level subdomains are a pain to get wildcard certificates for, so we should never use them, not even in scenarios where we aren't really using them.